helidon-linker/src/main/java/io/helidon/linker/util/Log.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.linker.util;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Simple, centralized logging.
+ */
+public abstract class Log {
```

Comment:
Why a custom `Log` abstraction? We should use `JUL` in our programs.

Reply:
Because this needs to integrate with the maven log mechanism, which is not JUL. See `MavenLogWriter`.

Reply:
Ok